mongo/gridfs_download_stream.go

```diff
@@ -14,7 +14,6 @@ import (
 	"time"
 
 	"go.mongodb.org/mongo-driver/bson"
-	"go.mongodb.org/mongo-driver/mongo"
 )
 
 // ErrMissingChunk indicates that the number of chunks read from the server is
```

Comment:
The docs for these errors should be updated to indicate they're only returned by GridFS operations.